src/main/java/com/jackshenorion/smarts/util/io/IOUtil.java

```diff
@@ -525,7 +525,9 @@ public static void writeAndClose(File file, byte[] data) {
 
 	public static void writeAndClose(File file, String data) {
 		try {
-			writeAndClose(new FileWriter(file), data);
+			if (true) { // for findbugs
+				writeAndClose(new FileWriter(file), data);
```

Comment:
**Reporter**: FINDBUGS
**Rule**: DM_DEFAULT_ENCODING
**Severity**: INFO
**File**: src/main/java/com/jackshenorion/smarts/util/io/IOUtil.java L529
**Source**: com.jackshenorion.smarts.util.io.IOUtil

Reliance on default encoding


<p> Found a call to a method which will perform a byte to String (or String to byte) conversion, and will assume that the default platform encoding is suitable. This will cause the application behaviour to vary between platforms. Use an alternative API and specify a charset name or Charset object explicitly.  </p>


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-308017806>*